src/test/java/basic/Exercise03Test.java

```diff
@@ -0,0 +1,18 @@
+package basic;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class Exercise03Test {
+
+    private final Exercise03 exercise03 = new Exercise03();
+
+    @Test
+    void factorizePrime() {
```

Comment:
Anh quên comment ở 2 pr kia, fix luôn

Có thể config IDE để gen tên hàm như thế này nha

```suggestion
    void shouldFactorizePrime_OK() {
```